lib/bpf/bpf_test.go

```diff
@@ -211,23 +208,28 @@ func (s *Suite) TestObfuscate(c *check.C) {
 	select {
 	case <-doneContext.Done():
 	case <-time.After(10 * time.Second):
-		c.Fatalf("Timed out waiting for an event.")
+		t.Fatalf("Timed out waiting for an event.")
 	}
 
 }
 
 // TestScript checks if execsnoop can capture what a script executes.
-func (s *Suite) TestScript(c *check.C) {
+func TestScript(t *testing.T) {
```

Comment:
This test and `TestObfuscate` have a lot of common logic. I think we could combine them. I'm just not sure if doing this as a part of this PR is a good idea.

Reply:
Yeah, let's do that in another PR. I think we can do a lot of cleanup to make our BPF tests much cleaner.